tests/testthat/test-bug-501-rectangles.R

```diff
@@ -0,0 +1,13 @@
+
+test_that("Edges stop at outside of rectangle node", {
+  rectangle_edges <- function() {
+    set.seed(1234)
+    g <- erdos.renyi.game(5, 6, 'gnm', directed = TRUE)
```

Comment:
The tests will be stabler in the long term if we added an explicit construction of `g`, and also for the layout, here and for the other plotting tests. This feels out of scope for this buglet because we seem to lack the necessary infrastructure to do this efficiently.

Reply:
Sorry, it's not clear to me if you're asking me to change the test or to drop it.  If you'd like everything explicit I can do that.

Reply:
If it's kept this way, `erdos.renyi.game` should be replaced with `sample_gnm` (as the former is now deprecated).